lib/tasks/email_checkout_reminder.rake

```diff
@@ -1,17 +1,17 @@
 desc 'Send email reminder about upcoming checkout'
 task email_checkout_reminder: :environment do
-  if AppConfig.first.upcoming_checkout_email_active?
-    # get all reservations that start today
-    upcoming_reservations = Reservation.upcoming
-    Rails.logger.info "Found #{upcoming_reservations.size} reservations that \
-      start today. Sending reminder emails..."
-    upcoming_reservations.each do |upcoming_reservation|
-      Rails.logger.info "Sending reminder for upcoming reservation:\n \
-        #{upcoming_reservation.inspect}"
-      UserMailer.reservation_status_update(upcoming_reservation).deliver
-    end
-  else
-    Rails.logger.info 'Upcoming check out emails are not sent by admin. \
-      Please change the application settings if you wish to send them.'
+  email_checkout_reminder
+end
+
+def email_checkout_reminder
+  return unless AppConfig.first.upcoming_checkout_email_active?
```

Comment:
Is there a reason why you got rid of the alternative log message (if they're not configured to be sent)? You kept them for check-in reminders, and I think it would be good to keep them here as well. As I understand it, guard clauses are preferred when the entire method executes only under a given condition. In this case, since we have code we'd like to execute if the setting is off, we would prefer an `if ... else` block. Does that make sense?